test/deprecates.jl

```diff
@@ -0,0 +1,37 @@
+@testset "Deprecations" begin
+    @testset "Syslog" begin
+        if Sys.isunix()
```

Comment:
Maybe have the `if` surround the testset so that this doesn't show up in the final count?